publisher/internal/analytics/ethereum/analytics.go

```diff
@@ -60,6 +64,9 @@ func init() {
 	transferSig = convertToEventSignature(transferEventHeader)
 	burnSig = convertToEventSignature(burnEventHeader)
 	collectSig = convertToEventSignature(collectEventHeader)
+
+	stableCoins = append(stableCoins, strings.ToLower(addressUSDT), strings.ToLower(addressUSDC))
```

Comment:
This is an interesting choice for variable initialization. However, a more clear way would be to simply assign a new slice to this and the next variable.